FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -325,12 +327,12 @@ NS_SWIFT_NAME(RemoteConfig)
 ///                         nil if the key doesn't exist in the default config.
 - (nullable FIRRemoteConfigValue *)defaultValueForKey:(nullable NSString *)key;
 
-#pragma mark - Realtime
+#pragma mark - Real-time Config Updates
 
 /// Completion handler invoked by `addOnConfigUpdateListener` when there is an update to
 /// the config from the backend.
 ///
-/// @param configUpdate Information on which key's values have changed
+/// @param configUpdate (``FIRRemoteConfigUpdate``) Information on which key's values have changed.
```

Comment:
If double backticks don't link to the method in our docs, I'd just use single so it renders as code.

```
@param configUpdate An instance of `FIRRemoteConfigUpdate` with information on which keys' values have changed.
```

Reply:
Changing to single back-ticks for now